packages/dropdown/src/FzIconDropdown.vue

```diff
@@ -0,0 +1,24 @@
+<template>
+  <FzDropdown v-bind="props" ref="dropdown">
+    <template #opener>
+      <FzIconButton :iconName="iconName" @click="dropdown?.open()" :variant="buttonVariant" />
```

Comment:
what if you provide the open function from the opener slot? It would make it much more handy

Reply:
makes sense. I think I will keep the exposed "open" function anyway as an escape hatch for users that may need that